spec/controllers/admin/spam_controller_spec.rb

```diff
@@ -18,21 +16,29 @@
       end
     end
 
-    context "when logged in as admin without correct authorization" do
-      xit "redirects with notice" do
-        fake_login_admin(admin)
-        get :index, params: { reviewed: false, approved: false }
+    %w[board communications translation tag_wrangling docs support open_doors].each do |role|
+      context "when logged in as an admin with #{role} role" do
+        let(:admin) { create(:admin, roles: [role]) }
 
-        it_redirects_to_with_error(root_url, "Sorry, only an authorized admin can access the page you were trying to reach.")
+        it "redirects with notice" do
```

Comment:
Just one last thing! It should probably say `redirects with error` here and on line 66.

Reply:
done